android/hello_sdl_android/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true"
```

Comment:
We can do that in Java Suite. But what happens if we change `http` to `https` for URL and the server was not setup to support `https`? will it be acceptable if the request fails in that case?

Reply:
That's my point, it will on iOS guaranteed, with no workarounds. So do we want things to work on Android that don't on iOS? I think we should align the platforms here to the most restrictive.

Reply:
Agreed

Reply:
removed

Reply:
please make sure that it is removed from the test `AndroidManifest.xml ` as well